services/api/src/schema.ts

```diff
@@ -42,6 +41,7 @@ export const countryCodeEnum = pgEnum("country_code", ["AND", "ARE", "AFG", "ATG
 export const ageGroupEnum = pgEnum("age_group", ["8-15", "16-24", "25-34", "35-44", "45-54", "55-64", "65+"]);
 export const sexEnum = pgEnum("sex", ["F", "M", "X"]);
 
+/*
 export const phoneCountryCodeEnum = pgEnum("phone_country_code", [
```

Comment:
By default an enum should not contain duplicate values.

```phoneCountryCodeEnum``` currently causes an error during the Flyway migration since I updated Flyway's version:

```
ERROR: Script V0001__optimal_stark_industries.sql failed
-------------------------------------------------
SQL State  : 23505
Error Code : 0
Message    : ERROR: duplicate key value violates unique constraint "pg_enum_typid_label_index"
  Detail: Key (enumtypid, enumlabel)=(17395, 61) already exists.
Location   : /flyway/sql/V0001__optimal_stark_industries.sql (/flyway/sql/V0001__optimal_stark_industries.sql)
Line       : 1
Statement  : Run Flyway with -X option to see the actual statement causing the problem

Caused by: Script V0001__optimal_stark_industries.sql failed
```

Reply:
I will remove the duplicate values from it

Reply:
This enum had been revised and placed back in the code.